docs/change-log.md

```diff
@@ -2,6 +2,16 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update January 20th, 2023
+
+- **New:** Humanitec's [Artefacts Service](https://api-docs.humanitec.com/#tag/ArtefactVersion) in addition to supporting branch and tag refs now also supports PR refs. 
```

Comment:
I'd stay in line with the docs here and refer to "Artefact API"

My suggestion would be something like: "The [Artefact Version API](https://api-docs.humanitec.com/#tag/ArtefactVersion) now supports PR refs. Next to branch- and tag refs artefact versions build on a PR branch can be pushed by users"